docs/maintenance.rst

```diff
@@ -148,3 +148,16 @@ This section contains useful links for maintainers and collaborators:
 
 - `Future of icalendar, looking for maintainer #360 <https://github.com/collective/icalendar/discussions/360>`__
 - `Comment on the Plone tests running with icalendar <https://github.com/collective/icalendar/pull/447#issuecomment-1277643634>`__
+
```

Comment:
yes, I like it. This is just about the changelog now.

Reply:
Nice